### ISSUE TITLE
A4046 - CE-2-AI0: form control is not associated with a text label

### ISSUE THREAD

**Issue by RolandCampbell** [RolandCampbell/ForgetMeNot1#3]
*Common Element*: Offers bar\\ *Issue link*: [AI0|http://aqa.usablenet.com/training-and-demo/roa/1642259846e_a92df74362b1807cbeeadc9ac5b9ee97/A4046/T10232/step1/details/view/AI0]\\ *Description*: form control is not associated with a text label\\ *WCAG Success Criterion*: 1.1.1\\ *Element*: select\\ *Severity*: high\\ *Complexity*: easy
